portfolio/src/main/webapp/script.js

```diff
@@ -57,4 +63,28 @@ function createListElement(text){
     listElement.setAttribute('class', 'list-group-item');
     listElement.setAttribute('onclick', 'activateListItem()')
     return listElement;
+}
+
+//convert ISO timezone to yyyy-mm-dd hh:mm
+function convertISOtoString(timestamp) {
+    console.log(timestamp);
+    timestamp = new Date(timestamp);
+    console.log(timestamp);
+
+    year = "" + timestamp.getFullYear();
+    month = padZero(timestamp.getMonth() + 1);
+    day = padZero(timestamp.getDate());
+    hour = padZero(timestamp.getHours());
+    minute = padZero(timestamp.getMinutes());
+
+    return year + "-" + month + "-" + day + " " + hour + ":" + minute;
+}
+
+// utility function to pad zero in front of month, day, hour, minute
+function padZero(text) {
+    text = "" + text;
```

Comment:
Oh, the javascript hack to do string formatting. That always make me feel some other strong typed language is so good.

Reply:
:)